docs/fastdds/dds_layer/domain/domainParticipant/domainParticipant.rst

```diff
@@ -1,5 +1,186 @@
++--------------------------+------------------------------+----------+
+| ``allocation_``          | ParticipantResourceLimitsQos | no       |
++--------------------------+------------------------------+----------+
+| ``properties_``          | PropertyPolicyQos            | no       |
++--------------------------+------------------------------+----------+
+| ``wire_protocol_``       | WireProtocolConfigQos        | no       |
++--------------------------+------------------------------+----------+
+| ``transport_``           | TransportConfigQos           | no       |
++--------------------------+------------------------------+----------+
+
+Refer to the detailed description of each QosPolicy class for more information about their usage and
+default values.
+
+The QoS value of a previously created :ref:`dds_layer_domainParticipant` can be modified using the
+:func:`set_qos()` method.
+Trying to modify an immutable QosPolicy on an already enabled :ref:`dds_layer_domainParticipant`
+will result on an error.
+In such case, no changes will be applied and the :ref:`dds_layer_domainParticipant` will keep its
+previous DomainParticipantQos.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DOMAINPARTICIPANTQOS
+   :end-before: //!
+
+
+.. _dds_layer_defaultDomainParticipantQos:
+
+Default DomainParticipantQos
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The default :ref:`dds_layer_domainParticipantQos` refers to the value returned by the
+:func:`get_default_participant_qos()` method on the :ref:`dds_layer_domainParticipantFactory` singleton.
+The special symbol :class:`PARTICIPANT_QOS_DEFAULT` can be used as QoS parameter on :func:`create_participant()`
+or :func:`set_qos()` methods to indicate that the current default :ref:`dds_layer_domainParticipantQos` should be used.
+
+When the system starts, the default :ref:`dds_layer_domainParticipantQos` is equivalent to the default constructed
+value :func:`DomainParticipantQos()`.
+The default :ref:`dds_layer_domainParticipantQos` can be modified at any time using the
+:func:`set_default_participant_qos()` method on the :ref:`dds_layer_domainParticipantFactory` singleton.
+Modifying the default :ref:`dds_layer_domainParticipantQos` will not affect already existing
+:ref:`dds_layer_domainParticipant` instances unless their QoS is modified with
+:func:`set_qos(PARTICIPANT_QOS_DEFAULT)`.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DEFAULT_DOMAINPARTICIPANTQOS
+   :end-before: //!
+
+:func:`set_default_participant_qos()` method also accepts the symbol :class:`PARTICIPANT_QOS_DEFAULT`
+as input parameter.
+This will reset the current default :ref:`dds_layer_domainParticipantQos` to the default constructed value
+:func:`DomainParticipantQos()`.
+
+.. literalinclude:: /../code/DDSCodeTester.cpp
+   :language: c++
+   :start-after: //DDS_CHANGE_DOMAINPARTICIPANTQOS_TO_DEFAULT
+   :end-before: //!
```

Comment:
This snippet is very much like the last part of the last one. I'd move the text to the previous paragraph and combine the code example

Reply:
On this one we are changing the default participant Qos, stored on the Factory. On the previous one we were changing the QoS of a Domain instance. They may sound similar,but are quite different in purpose. That's why I keep them separated.